app/build.gradle.kts

```diff
@@ -70,11 +67,8 @@ dependencies {
     implementation(libs.timber)
 
     implementation(libs.work.runtime)
-    implementation(libs.hilt.work)
-    implementation(libs.hilt.common)
 
     androidTestImplementation(libs.android.test.junit4)
```

Comment:
do we need this?

Reply:
Yes we do.. I moved them to the hilt convention plugin.